docs/content/development/datasketches-aggregators.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc_page
+---
+
+## DataSketches aggregator
+Druid aggregators based on [datasketches]()http://datasketches.github.io/) library. You would ingest one or more metric columns using either `sketchBuild` or `sketchMerge` aggregators. Then, at query time, you can use `sketchMerge` with appropriate post aggregators described below. Note that sketch algorithms are approxiate, see details in the [datasketches doc](http://datasketches.github.io/docs/theChallenge.html).
+
+#### aggregators to use at ingestion time
+
+##### non-sketch input data
+You can ingest data to druid and build the sketch objects which are later used at the time of querying to compute uniques.
```

Comment:
I am a little confused by your usage of the word 'sketch'. In my understanding a 'sketch' in druid is any fancy column that can be merged like the hyperUniques and approxHistograms do DataSketches work on those as well? Or are DataSketches their own things that live next to the hyperUnique sketch? If it is the former that is really cool, you should elaborate on it; if it is the latter you should give DataSketches a more descriptive name.